### ISSUE TITLE
Trigger kubelet restart after a bbr restore

### ISSUE THREAD

**Issue by jaimegag** [cloudfoundry-incubator/kubo-release#288]
**Is your feature request related to a problem? Please describe.**
It is due to a problem:
If we `bbr restore` to a newly deployed cluster, since the names of the nodes are now different and a previous deployment/etcd database gets restored, once it gets restored the worker nodes disappear
If we restart them manually everything proceeded to get scheduled correctly.
This is specially important on OpenStack where the node names are the VM UUIDs so they are always going to be different when we restore in a newly deployed cluster.

**Describe the solution you'd like**
Trigger a restart in the workers' kubelet after a `bbr restore`.

**Describe alternatives you've considered**
As a temporary workaround we could restart kubelets manually

**Additional context**
N/A

**Comment by glestaris** [comment]
hey @jaimegag, I just came across this and it's timely. We are going to explore restore to a new CFCR cluster soon: https://www.pivotaltracker.com/story/show/164052833.

cc @aguedes-pivotal

**Comment by jaimegag** [comment]
@glestaris Great!, let me know. The customer is asking again if we could introduce the restart of the workers' kubelet after a `bbr restart`

**Comment by gmrodgers** [comment]
hey @jaimegag 
We are looking into this now as part of the story that George linked above.  We are curious to know if you took any other steps as part of your migration in order to get your cluster to be healthy again.

In the scenario we are testing, we did the following:
1. Deployed two clusters of the same topology
1. Deployed a [smorgasboard](https://github.com/cloudfoundry-incubator/kubo-ci/blob/master/specs/smorgasbord.yml) onto one of the clusters
1. Took a backup of that cluster
1. Restored this artifact onto the other cluster
1. Restarted the worker node and deleted the old node

We found that many `kube-system` pods are in the CrashLoopRunOff state after restore, as so:
```
kubectl --namespace=kube-system get pods
NAME                                    READY   STATUS             RESTARTS   AGE
heapster-85647cf566-74nc4               1/1     Running            0          19h
kube-dns-7559c96fc4-cgvr5               1/3     CrashLoopBackOff   608        19h
kubernetes-dashboard-5f4b59b97f-rvf42   0/1     CrashLoopBackOff   230        19h
metrics-server-555d98886f-qn2md         1/1     Running            0          19h
monitoring-influxdb-cdcf4674-246r7      1/1     Running            0          19h
```

Did you have similar issues when you migrated?  What was the entire flow of your migration?  We want to avoid duplicating effort if you've already experienced these issues.

Thanks
Glen && @alamages
